manifest.json

```diff
@@ -404,6 +404,73 @@
                 "PublishLayerVersion"
             ]
         },
+        "MediaConvert": {
+            "source": "https://raw.githubusercontent.com/aws/aws-sdk-php/${LATEST}/src/data/mediaconvert/2017-08-29/api-2.json",
+            "documentation": "https://raw.githubusercontent.com/aws/aws-sdk-php/${LATEST}/src/data/mediaconvert/2017-08-29/docs-2.json",
+            "pagination": "https://raw.githubusercontent.com/aws/aws-sdk-php/${LATEST}/src/data/mediaconvert/2017-08-29/paginators-1.json",
+            "api-reference": "https://docs.aws.amazon.com/mediaconvert/latest/apireference",
+            "methods": [
+                "CancelJob",
+                "CreateJob",
+                "DescribeEndpoints",
+                "GetJob",
+                "ListJobs"
+            ],
+            "patches": {
+                "source": [
+                    {
+                        "path": "/operations/CancelJob/endpointdiscovery",
+                        "op": "add",
+                        "value": {
+                            "required": true
+                        }
+                    },
+                    {
+                        "path": "/operations/CreateJob/endpointdiscovery",
+                        "op": "add",
+                        "value": {
+                            "required": true
+                        }
+                    },
+                    {
+                        "path": "/operations/GetJob/endpointdiscovery",
+                        "op": "add",
+                        "value": {
+                            "required": true
+                        }
+                    },
+                    {
+                        "path": "/operations/ListJobs/endpointdiscovery",
+                        "op": "add",
+                        "value": {
+                            "required": true
+                        }
+                    }
+                ],
+                "documentation": [
+                    {
+                        "path": "/shapes/__stringMin1/refs/CaptionDescription$CaptionSelectorName",
+                        "op": "replace",
+                        "value": "Specifies which \"Caption Selector\":#inputs-caption_selector to use from each input when generating captions. The name should be of the format \"Caption Selector N\", which denotes that the Nth Caption Selector will be used from each input."
```

Comment:
The remaining HTML detection complains when the result contains `<`. And if the goal is to provide markdown without any embedded HTML, it would indeed require wrapping those in a code span.

Reply:
From what I can see 100% of the mediaconvert's documentation is plain text. 

I suggest disabling the HTML to markdown conversion (or at least disabling the final exception about remaining `<`).

I aim to minimize the patching section and keep documentation in sync as much as possible.

Reply:
What would be the best way to disabling this conversion ?

Reply:
I will actually wait for the conclusion on #1418 before working on this, as it might solve the issue entirely if the final decision is to remove the long documentation.

Reply:
I solved this by enclosing those tags in backticks to make them markdown codeblocks and then stripping markdown codeblocks before detecting remaining tags as they are not HTML tags in that location in markdown.